util/spell.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if ! which -s aspell; then
```

Comment:
preference to: `hash aspell 2>/dev/null || { echo ' ... '; exit 1 }`

notably, we need to provide an exit code on L5; by default `exit` will return 0 which implies success; that's not the case here. I believe 127 is common for "file not found" returns, but frankly I'm cool with anything other than 0.